topics/dev/tutorials/conda/slides.html

```diff
@@ -845,7 +849,18 @@
 
 ## Pinning Libraries - Use jinja
 
-> If your package links dynamically against a particular library, it is **often necessary to pin the version against which it was compiled, in order to avoid ABI incompatibilities**. Instead of hardcoding a particular version in the recipe, we use jinja templates to achieve this. This **helps ensure that all bioconda packages are binary-compatible with each other**. For example, bioconda provides an environment variable `CONDA_BOOST` that contains the current major version of Boost.
+> If your package links dynamically against a particular library, it is **often necessary to pin the version against which it was compiled, in order to avoid ABI incompatibilities**. Instead of hardcoding a particular version in the recipe, we use jinja templates to achieve this. This **helps ensure that all bioconda packages are binary-compatible with each other**. For example, bioconda provides an environment variable `CONDA_ZLIB` that contains the current major version of Zlib.
+
+
+```yaml
+{% raw %}
+requirements:
+  build:
+    - zlib {{ CONDA_ZLIB }}*
```

Comment:
if you use zlib you most likely also include gcc/llvm ... not sure if this is disturbing for the user ... but the recipe as it is will most likely not exists in the wild.